File: ass-ade-v1.1/tests/fixtures/rebuilt-out/bridges/swift/Sources/AssAdeBridge/a2_mo_composites/PythonBridgeClient.swift
import Foundation

final class PythonBridgeClient {
    private let start: URL

    init(start: URL) {
        self.start = start
    }

    func manifest() throws -> BridgeManifestSummary {
        try loadBridgeManifest(start: start)
    }

    func manifestPayload() throws -> String {
        try loadManifestPayload(start: start)
    }
}

File: tests/v11/fixtures/rebuilt-out/bridges/rust/src/a2_mo_composites/python_bridge_client.rs
use std::path::PathBuf;

use crate::a0_qk_constants::bridge_contract::BridgeManifestSummary;
use crate::a1_at_functions::manifest_loader::{find_repo_root, load_manifest, load_manifest_payload};

pub struct PythonBridgeClient {
    repo_root: PathBuf,
}

impl PythonBridgeClient {
    pub fn new(start: PathBuf) -> Result<Self, String> {
        Ok(Self {
            repo_root: find_repo_root(start)?,
        })
    }

    pub fn manifest(&self) -> Result<BridgeManifestSummary, String> {
        load_manifest(self.repo_root.clone())
    }

    pub fn manifest_payload(&self) -> Result<String, String> {
        load_manifest_payload(self.repo_root.clone())
    }
}
